subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -368,7 +365,8 @@ config BT_CTLR_SCHED_ADVANCED
 		   (BT_CONN || \
 		    (BT_CTLR_ADV_EXT && (BT_CTLR_ADV_AUX_SET > 1)) || \
 		    BT_CTLR_ADV_ISO)
-	default y if BT_CENTRAL || (BT_BROADCASTER && BT_CTLR_ADV_EXT) || BT_CTLR_ADV_ISO
+	default y if (BT_CENTRAL || (BT_BROADCASTER && BT_CTLR_ADV_EXT) || BT_CTLR_ADV_ISO) && \
+		     BT_LL_SW_LLCP_LEGACY
```

Comment:
Link a GH issue to fix the missing support for advanced scheduling in new LLCP

Reply:
Done